src/app/component/core/header-new/header-new.component.ts

```diff
@@ -40,8 +42,10 @@ export class HeaderNewComponent implements OnInit {
               private userService: UserService,
               private achievementService: AchievementService,
               private habitStatisticService: HabitStatisticService,
-              private languageService: LanguageService) {
-}
+              private languageService: LanguageService,
+              private clickSearch: SearchClickService) {
+    this.clickSearch.emitter.subscribe(signal => this.isSearchClicked = signal);
```

Comment:
Please, write this function in ngOnInit hook, if it is possible

Reply:
fixed